runtime/kusama/src/weights/runtime_parachains_paras_inherent.rs

```diff
@@ -70,28 +74,30 @@ impl<T: frame_system::Config> runtime_parachains::paras_inherent::WeightInfo for
 	// Storage: Hrmp HrmpWatermarks (r:0 w:1)
 	// Storage: Paras Heads (r:0 w:1)
 	fn enter_variable_disputes(v: u32, ) -> Weight {
-		(202_257_000 as Weight)
-			// Standard Error: 2_000
-			.saturating_add((311_000 as Weight).saturating_mul(v as Weight))
-			.saturating_add(T::DbWeight::get().reads(23 as Weight))
-			.saturating_add(T::DbWeight::get().writes(14 as Weight))
+		(282_901_000 as Weight)
+			// Standard Error: 13_000
+			.saturating_add((49_184_000 as Weight).saturating_mul(v as Weight))
```

Comment:
This value is consistent with the baseline benchmarks for signature verification https://github.com/paritytech/substrate/pull/10414/files#diff-54513fbed8656a7d28423c4c68a9cfd6b9daa7be7cdf623ba1a7b496e21f9edeR77-R80